README.md

```diff
@@ -18,9 +18,11 @@
 
 ### Developing the application
 
-1. Clone the repository with `git clone --resuce-submodules git@github.com:Metatavu/time-bank-api-v2.git`
-2. Start a MySQL database with Docker or whatever
-3. Ask for environment variables from someone who has worked with the project
-4. Run `quarkus dev`
-5. Look up [Quarkus guides](https://quarkus.io/guides/)
-6. **Enjoy!**
\ No newline at end of file
+1. Clone the repository with `git clone --recurse-submodules git@github.com:Metatavu/time-bank-api-v2.git`
+2. If you are using Linux, check Linux post-install guide for Docker
+3. Gradle JVM should be at least SKD/JDK 11
+4. Build the application with `./gradlew generateApiSpec` & `./gradlew generateApiClient`
```

Comment:
To be precise, this isn't for building the application and also these two tasks should be made dependent for the actual build task, if they aren't already. e.g. https://github.com/Metatavu/time-bank-api-v2/blob/490653164b7cfaddb7208e561f593d7714bab52d/build.gradle.kts#L116-L122